main.py

```diff
@@ -11,7 +11,7 @@ def search(request: Request):
     return templates.TemplateResponse('search.html', {'request': request})
 
 @app.get('/search')
-def search_flights(from_city: str = Query(...), to_city: str = Query(...), date: str = Query(...)):
-    params = {'from': from_city, 'to': to_city, 'date': date}
+def search_flights(fly_from_city: str = Query(...), to_city: str = Query(...), date: str = Query(...)):
+    params = {'fly_from': fly_from_city, 'to': to_city, 'date': date}
```

Comment:
How do you know this is the issue?

Reply:
The error message indicates that the API call is not correctly specifying the departure location. Looking at the Tequila API documentation, the parameter for the departure location is 'fly_from', not 'from'. Therefore, changing 'from' to 'fly_from' in the parameters dictionary should resolve the issue.